[DOCS 6486] - Adding DigitalOcean sponsorship blog post (#5448)
jenkins-infra/jenkins.io: content/blog/2022/09/2022-09-18-digital-ocean-sponsorship.adoc

We are excited to announce that the relationship between DigitalOcean and Jenkins continues to grow!
We want to thank DigitalOcean for providing such incredible support to the Jenkins project.

DigitalOcean has provided a substantial donation of $18,000 to the Jenkins project.
This is in addition to their previous sponsorship link:https://www.jenkins.io/blog/2022/05/04/DigitalOcean/[described here]. 
Combining these, DigitalOcean has granted over *$20,000* to the Jenkins project.
The funding aspect is clear, but there are several other benefits to working with DigitalOcean in this capacity.
Jenkins has access to more resources than ever before, such as utilizing the DigitalOcean platform to continue development and share progress made.
We can dedicate our time and resources more fully, allowing users to engage and make an impact.

This sponsorship also increases the impact that we can have as a community, when it comes to ongoing projects and events like link:https://hacktoberfest.com/[Hacktoberfest], which DigitalOcean organizes and hosts.
DigitalOcean is proving their commitment to open source, which matches our desire to improve Jenkins in any way and by any means.

> If you are new to link:https://digitalocean.com/[DigitalOcean] and would like to learn more, use our promo code `DOJENKINS100` to get $100 credit to use over your first 60 days (this offer is only valid for new accounts).

Reviewer: ```suggestion
If you are new to link:https://digitalocean.com/[DigitalOcean] and would like to learn more, use our promo code `DOJENKINS100` to get $100 credit to use over your first 60 days (only valid for new accounts).
```

Author: thanks for the feedback on this one too @lemeurherve, I just updated things accordingly!

---
layout: post
title: "Digital Ocean and Jenkins partnership continues to grow!"
tags:
- jenkins
- digitalocean
- community
author: kmartens27
description: "DigitalOcean increases Jenkins sponsorship"
opengraph:
  image: /images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png
---

> We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!

Reviewer 2: Optional suggestion. 
```suggestion
We are excited to announce that the relationship between DigitalOcean and Jenkins continues to grow!
```

Author: same idea,  much better and smoother to read, thanks so much @kellie-freeman !!

> Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.

Reviewer 2: I wasn't sure if we should make it more personal by using "We"? 
```suggestion
We want to thank DigitalOcean for providing such incredible support to the Jenkins project.
```

Author: I definitely like that better @kellie-freeman, much smoother and natural to read.

> image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]

Reviewer 2: You may need an extra line break here for the image to render properly. If not, please disregard!
```suggestion

image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]
```

Author: moved it to the top of the page instead, while including an extra break between, thanks so much @kellie-freeman

> 
> We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
> Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.
> image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]

Reviewer: I would put an empty line before the logo
```suggestion

We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.

image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]
```

or even put it at the top of the article:
```suggestion
image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]

We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.
```

DigitalOcean has provided a substantial donation of $18,000 to the Jenkins project.
This is in addition to their previous sponsorship link:https://www.jenkins.io/blog/2022/05/04/DigitalOcean/[described here]. 
Combining these, DigitalOcean has granted over *$20,000* to the Jenkins project.
The funding aspect is clear, but there are several other benefits to working with DigitalOcean in this capacity.
Jenkins has access to more resources than ever before, such as utilizing the DigitalOcean platform to continue development and share progress made.
We can dedicate our time and resources more fully, allowing users to engage and make an impact.

> This sponsorship also increases the impact that we can have as a community, when it comes to ongoing projects and events like Hacktoberfest, which DigitalOcean organizes and hosts.

Reviewer: ```suggestion
This sponsorship also increases the impact that we can have as a community, when it comes to ongoing projects and events like link:https://hacktoberfest.com/[Hacktoberfest], which DigitalOcean organizes and hosts.
```

Author: Great call out @lemeurherve, that will be much better!

DigitalOcean is proving their commitment to open source, which matches our desire to improve Jenkins in any way and by any means.

If you are new to link:https://digitalocean.com/[DigitalOcean] and would like to learn more, use our promo code `DOJENKINS100` to get $100 credit to use over your first 60 days. 

> This offer is only valid for new accounts. 

Reviewer 3: (nitpicking suggestion)
The final phrase could be put between "(", as a side note.

Author: definitely agree here @jmMeessen , I've gone ahead and added it to the end of the previous sentence and surrounded it with parentheses.
